hs_collection_resource/views.py

```diff
@@ -0,0 +1,106 @@
+import json
+import logging
+
+from django.http import HttpResponseRedirect, HttpResponseNotFound, HttpResponse, JsonResponse
+
+from hs_core import hydroshare
+from hs_core.views import _set_resource_sharing_status
```

Comment:
@zhiyuli The client.post call to set the resource flag looks correct to me. The posting of data as a dictionary {'t': 'make_public'} seems to be correct as per the Django documentation. The only thing I am not sure of is the user of 'HTTP_REFERER' in that post call.